New: Add tango-admin docs (#333)
tango-controls/tango-doc: source/administration/maintenance/tango-admin.rst

.. _tango_admin:

> Tango admin
> =================

Reviewer: ```suggestion
Tango Admin utility
===================
```

:audience:`administrators`

> The Tango Database can also be maintained using a command-line interface. Using this utility, not all the database features are interfaced. Only the features needed for the Debian packaging have been implemented.
> 
> This means:

Reviewer: ```suggestion
The Tango Database can be maintained using a command-line interface with :program:`tango_admin` tool. The following features are available:
```

* ping the database server
* check if a device is defined in DB
* check if a server is defined in DB
* create a server in DB
* delete a server from the DB
* create a property in DB
* delete a property from DB

> Using Tango Admin
> -----

Reviewer: ```suggestion
Using tango_admin
-----------------
```
